custom/ilsgateway/tanzania/handlers/keyword.py

```diff
@@ -27,6 +27,9 @@ def sql_location(self):
         if location:
             return location.sql_location
 
+        if not self.args:
+            return
```

Comment:
@kkrampa do you mean that any time self.args is blank, it will cause that error message to be sent?